### PR TITLE
Kubelet: Fix the description of MaxContainers kubelet flag.

### PR DESCRIPTION
**Pull request by Random-Liu**
Found this during code review.

The default number has been changed to `-1` and `1`. https://github.com/kubernetes/kubernetes/blob/82c488bd6ed00b7417ee32bf468d70f4dd18c25a/pkg/apis/componentconfig/v1alpha1/defaults.go#L279-L285
@yujuhong 

/cc @saad-ali This PR fixed incorrect doc.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/36471)

### CODE REVIEW

**Comment by saad-ali** [comment]
Ack

File: cmd/kubelet/app/options/options.go [hunk 1]
```diff
@@ -153,9 +153,9 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
...
 	fs.MarkDeprecated("minimum-container-ttl-duration", "Use --eviction-hard or --eviction-soft instead. Will be removed in a future version.")
-	fs.Int32Var(&s.MaxPerPodContainerCount, "maximum-dead-containers-per-container", s.MaxPerPodContainerCount, "Maximum number of old instances to retain per container.  Each container takes up some disk space.  Default: 2.")
+	fs.Int32Var(&s.MaxPerPodContainerCount, "maximum-dead-containers-per-container", s.MaxPerPodContainerCount, "Maximum number of old instances to retain per container.  Each container takes up some disk space.  Default: 1.")
 	fs.MarkDeprecated("maximum-dead-containers-per-container", "Use --eviction-hard or --eviction-soft instead. Will be removed in a future version.")
-	fs.Int32Var(&s.MaxContainerCount, "maximum-dead-containers", s.MaxContainerCount, "Maximum number of old instances of containers to retain globally.  Each container takes up some disk space.  Default: 100.")
+	fs.Int32Var(&s.MaxContainerCount, "maximum-dead-containers", s.MaxContainerCount, "Maximum number of old instances of containers to retain globally.  Each container takes up some disk space. To disable, set to negative number.  Default: -1.")
```
Comment (yujuhong) [inline cmd/kubelet/app/options/options.go]: s/negative/a negative

Comment (Random-Liu) [inline cmd/kubelet/app/options/options.go]: ACK.

**Comment by yujuhong** [comment]
LGTM

**Pull request merged**